libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -2329,7 +2329,7 @@ bool AP_AHRS::attitudes_consistent(char *failure_msg, const uint8_t failure_msg_
         // check roll and pitch difference
         const float rp_diff_rad = primary_quat.roll_pitch_difference(dcm_quat);
         if (rp_diff_rad > ATTITUDE_CHECK_THRESH_ROLL_PITCH_RAD) {
-            hal.util->snprintf(failure_msg, failure_msg_len, "DCM R/P differs from current AHRS by %d deg", (int)degrees(rp_diff_rad));
+            hal.util->snprintf(failure_msg, failure_msg_len, "DCM: R/P inconsistent. Wait or check sensors");
```

Comment:
"DCM: Roll/Pitch difference. Wait or check sensors"
for consistency with EKF2 and EKF3

Reply:
again, disagree with your suggestion